test/bdd/go.mod

```diff
@@ -8,6 +8,7 @@ go 1.14
 
 require (
 	github.com/Microsoft/hcsshim v0.8.7 // indirect
+	github.com/btcsuite/btcutil v1.0.1
```

Comment:
why is this line added in this PR?

Reply:
Because someone missed to add it. I’ve just added it into my PR.